packages/expo-video/src/VideoView.web.tsx

```diff
@@ -111,6 +111,7 @@ export const VideoView = forwardRef((props: { player?: VideoPlayer } & VideoView
   return (
     <video
       controls={props.nativeControls ?? true}
+      playsInline={props.playsInline ?? true}
```

Comment:
Shouldn't we take `allowsFullscreen` into account here? If it's allowed, maybe the `playsInline` should stay absent so it defaults to browser's default behavior.

Reply:
I think playsInline and allowsFullscreen can co-exist. It's a hint for the player to show the full screen button, but I am not 100% sure

Reply:
I think they can/should coexist, but I would force `playsInline` to true if `allowsFullscreen` is false.

Reply:
I agree with @aleqsio, we should force it to true when allowsFullscreen is false